kb/2023-08-11/using_netapp_third_party_storage.md

```diff
@@ -0,0 +1,154 @@
+   - `kubectl apply -f deploy/bundle_post_1_25.yaml`
+   The operator, deployment, and replicasets can be verified with:
+   - `kubectl get all -n trident`
+4.c. Create the TridentOrchestrator and install Trident
+   - `kubectl create -f deploy/crds/tridentorchestrator_cr.yaml`
+   This will take a while (a few minutes); you can check progress with
+   - `kubectl desribe torc trident`
+   The last few lines of output will tell you whether the installation
+   has completed successfully.  When it has finished, the last few
+   lines should look something like this:
+
+```
+Events:
+    Type Reason Age From Message ---- ------ ---- ---- -------Normal
+    Installing 74s trident-operator.netapp.io Installing Trident Normal
+    Installed 67s trident-operator.netapp.io Trident installed
+```
+
+4.d. Verify the installation
+   - `kubectl get pods -n trident`
+   - `./tridentctl -n trident version`
+
+5. Install a backend.
+   This part is specific to Harvester.
+5.a.    Put the following into a text file, for example /tmp/backend.yaml
+
+```
+version: 1
+backendName: default_backend_san
+storageDriverName: ontap-san-economy
+managementLIF: 172.19.97.114
+svm: default_backend
+username: admin
+password: password1234
+labels:
+  name: default_backend_san
+```
+    The LIF IP address, username, and password of this file
+    should be replaced with the management LIF and credentials
+    for the ONTAP system.
+
+5.b. Create the backend
+    - `./tridentctl create backend -f /tmp/backend.yaml -n trident`
+
+5.c. Check that it is created
+    - `./tridentctl get backend -n trident`
+
+6. Define a StorageClass and SnapshotClass
+6.a. Put the following into a file, for example /tmp/storage.yaml
+
+```
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: ontap-san-economy
+provisioner: csi.trident.netapp.io
+parameters:
+  selector: "name=default_backend_san"
+---
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshotClass
+metadata:
+  name: csi-snapclass
+driver: csi.trident.netapp.io
+deletionPolicy: Delete
+```
+
+6.b. Apply the definitions:
+    - `kubectl apply -f /tmp/storage.yaml`
+
+7. Enable multipathd
+   The following is required to enable multipathd.
+   This must be done on every node of the Harvester cluster.
+   The preceding steps should only be done once on a single node.
+7.a. Create this file in `/oem/99_multipathd.yaml`:
+```
+stages:
+   default:
+     - name: "Setup multipathd"
+       systemctl:
+         enable:
+          - multipathd
+         start:
+          - multipathd
+```
+7.b. Enable multipathd
+    The above file will take effect on the next reboot of the node;
+    multipathd can be enabled immediately without rebooting the node
+    using the following commands:
+    - `systemctl enable multipathd`
+    - `systemctl start multipathd`
+
+After the above steps, the `ontap-san-economy` storage class
+should be available for use when creating a volume for
```

Comment:
can we please add info on either disabling mutlipathd for longhorn or additional steps for disabling multipathd support in ontap, plus reasons for the same.

Reply:
I think I've done this now.  We've discussed what I've done but any comments on the results will be appreciated